okapi-common/src/main/java/org/folio/okapi/common/logging/FolioLoggingContext.java

```diff
@@ -0,0 +1,70 @@
+package org.folio.okapi.common.logging;
+
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.config.plugins.Plugin;
+import org.apache.logging.log4j.core.lookup.StrLookup;
+
+/**
+ * This class should be used for storing context variables
+ * and use them in logging events.
+ */
+@Plugin(name = "FolioLoggingContext", category = StrLookup.CATEGORY)
+public class FolioLoggingContext implements StrLookup {
+
+  private static final String EMPTY_VALUE = "";
+
+  private static final String LOGGING_VAR_PREFIX = "folio_";
+
+  public static final String TENANT_ID_LOGGING_VAR_NAME = "tenantid";
+
+  public static final String REQUEST_ID_LOGGING_VAR_NAME = "requestid";
+
+  public static final String MODULE_ID_LOGGING_VAR_NAME = "moduleid";
+
+  public static final String USER_ID_LOGGING_VAR_NAME = "userid";
+
+  /**
+   * Lookup value by key.
+   *
+   * @param key the name of logging variable (e.g. requestId)
```

Comment:
Can you add whether null key is allowed?
The same for lookup(LogEvent, String)?